### PR TITLE
docs(how-to): connect to an Elastic Metal server remotely using windows MTA-2197

### PR DESCRIPTION
**Pull request by nedaehm**
### Your checklist for this pull request

- [ ] Check that the commit messages match our requested structure.
- [ ] Name your pull request according to the [contribution guidelines](../docs/CONTRIBUTING.md).

### Description
Please describe your pull request.

### CODE REVIEW

File: compute/elastic-metal/how-to/connect-to-server.mdx [hunk 1]
```diff
@@ -51,6 +52,35 @@ This page shows how to connect to your Scaleway Elastic Metal server via SSH. Th
...
+
+  <Lightbox src="scaleway-configuration-page-microsoft-remote.webp" alt="" />
+
+4. Enter your Elastic Metal server's IP address in the **PC name** field and click **Add**. 
+
+5. Right click your newly added PC and select **Connect**.
+
+6. When prompted, enter `Administrateur` in the **Username** field and your password in the **Password** field. 
+
+7. Click **Continue**. Your are redirected to the Windows Server Manager.
```
Comment (nedaehm) [inline compute/elastic-metal/how-to/connect-to-server.mdx:80]:
```suggestion
7. Click **Continue**. You are redirected to the Windows Server Manager.
```

File: compute/elastic-metal/how-to/connect-to-server.mdx [hunk 2]
```diff
@@ -51,6 +52,35 @@ This page shows how to connect to your Scaleway Elastic Metal server via SSH. Th
 
 You are now connected to your Elastic Metal server.
 
+## How to connect from Windows using remote access
```
Comment (n-Arno) [inline compute/elastic-metal/how-to/connect-to-server.mdx:55]: How to connect to a Windows server using remote access

File: compute/elastic-metal/how-to/connect-to-server.mdx [hunk 3]
```diff
@@ -51,6 +52,35 @@ This page shows how to connect to your Scaleway Elastic Metal server via SSH. Th
 
 You are now connected to your Elastic Metal server.
 
+## How to connect from Windows using remote access 
+
+To connect to your Elastic Metal server from Windows, you need to download a remote access tool. For the sake of this tutorial, we use [Microsoft Remote Desktop](https://apps.apple.com/us/app/microsoft-remote-desktop/id1295203466?mt=12).
```
Comment (n-Arno) [inline compute/elastic-metal/how-to/connect-to-server.mdx:57]: To connect to your Elastic Metal server running windows,

Comment (nedaehm) [inline compute/elastic-metal/how-to/connect-to-server.mdx:57]:
```suggestion
To connect to your Elastic Metal server running Windows, you need to download a remote access tool. For the sake of this tutorial, we use [Microsoft Remote Desktop](https://apps.apple.com/us/app/microsoft-remote-desktop/id1295203466?mt=12). 
```

File: compute/elastic-metal/how-to/connect-to-server.mdx (hunk 2 above)
Comment (nedaehm) [inline compute/elastic-metal/how-to/connect-to-server.mdx:55]:
```suggestion
## How to connect to a Windows server using remote access 
```

File: compute/elastic-metal/how-to/connect-to-server.mdx [hunk 4]
```diff
@@ -51,6 +52,35 @@ This page shows how to connect to your Scaleway Elastic Metal server via SSH. Th
...
+## How to connect to a Windows server using remote access 
+
+To connect to your Elastic Metal server running Windows, you need to download a remote access tool. For the sake of this tutorial, we use [Microsoft Remote Desktop](https://apps.apple.com/us/app/microsoft-remote-desktop/id1295203466?mt=12). 
+
+1. [Create an Elastic Metal server](/compute/elastic-metal/how-to/create-server/) and choose a Windows image to run on it.
+
+  <Message type="important">
+
+    - Windows images are not available for the Aluminium range of Elastic Metal servers.
+    - Make sure you save your password securely after creating your Elastic Metal.
```
Comment (RoRoJ) [inline compute/elastic-metal/how-to/connect-to-server.mdx:64]:
```suggestion
    - Make sure you save your password securely after creating your Elastic Metal server.
```

File: compute/elastic-metal/how-to/connect-to-server.mdx [hunk 5]
```diff
@@ -51,6 +52,35 @@ This page shows how to connect to your Scaleway Elastic Metal server via SSH. Th
...
+
+  </Message>
+
+2. Download Microsoft Remote Desktop and launch it on your computer.
```
Comment (RoRoJ) [inline compute/elastic-metal/how-to/connect-to-server.mdx:68]: I'd probably link it again, for convenience?

File: compute/elastic-metal/how-to/connect-to-server.mdx [hunk 6]
```diff
@@ -51,6 +52,35 @@ This page shows how to connect to your Scaleway Elastic Metal server via SSH. Th
...
+
+4. Enter your Elastic Metal server's IP address in the **PC name** field and click **Add**. 
+
+5. Right click your newly added PC and select **Connect**.
+
+6. When prompted, enter `Administrateur` in the **Username** field and your password in the **Password** field. 
+
+7. Click **Continue**. You are redirected to the Windows Server Manager.
+
+You are now connected to your Elastic Metal server from Windows.
```
Comment (RoRoJ) [inline compute/elastic-metal/how-to/connect-to-server.mdx:82]:
```suggestion
You are now connected to your Elastic Metal server running Windows.
```

File: compute/elastic-metal/how-to/connect-to-server.mdx [hunk 7]
```diff
@@ -51,6 +52,35 @@ This page shows how to connect to your Scaleway Elastic Metal server via SSH. Th
...
+
 ## How to connect from Windows
```
Comment (RoRoJ) [inline compute/elastic-metal/how-to/connect-to-server.mdx:84]: I think I'd be tempted to have the order of things on this page be:

- How to connect from OSX (Mac) and Linux
- How to connect from Windows
- How to connect to an Elastic Metal server running Windows

...to me it makes a bit more sense that way, but not sure if it's just me?

File: compute/elastic-metal/how-to/connect-to-server.mdx (hunk 6 above)
Comment (n-Arno) [inline compute/elastic-metal/how-to/connect-to-server.mdx:82]: You are now connected to your Elastic Metal server running Windows.

File: compute/elastic-metal/how-to/connect-to-server.mdx [hunk 8]
```diff
@@ -51,6 +52,35 @@ This page shows how to connect to your Scaleway Elastic Metal server via SSH. Th
...
+1. [Create an Elastic Metal server](/compute/elastic-metal/how-to/create-server/) and choose a Windows image to run on it.
+
+  <Message type="important">
+
+    - Windows images are not available for the Aluminium range of Elastic Metal servers.
+    - Make sure you save your password securely after creating your Elastic Metal server.
+
+  </Message>
+
+2. Download Microsoft Remote Desktop and launch it on your computer.
```
Comment (nedaehm) [inline compute/elastic-metal/how-to/connect-to-server.mdx:68]:
```suggestion
2. [Download Microsoft Remote Desktop](https://apps.apple.com/us/app/microsoft-remote-desktop/id1295203466?mt=12) and launch it on your computer. 
```

File: compute/elastic-metal/how-to/connect-to-server.mdx [hunk 9]
```diff
@@ -83,13 +85,50 @@ To connect to your Elastic Metal server from Windows, you will need to use a sma
...
+
+  <Message type="note">
+
+  - [Download Microsoft Remote Desktop for OSX](https://apps.apple.com/us/app/microsoft-remote-desktop/id1295203466?mt=12)
+  - [Download Microsoft Remote Desktop for Linux]()
```
Comment (nedaehm) [inline compute/elastic-metal/how-to/connect-to-server.mdx:113]:
```suggestion
```

**Review by RoRoJ — APPROVED** [review]

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**